### ISSUE TITLE
Improved linking of project datasets

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/rdl-data#16]
support explicit linking, via project ID fields in all schema > enables linking independent of loss data (enable link across exp, hazard only for example). Current linkage occurs in loss dataset only

**Comment by matamadio** [comment]
Yes, being able to group all datasets by project would be useful.

**Comment by matamadio** [comment]
This is expected behaviour if we enable fuzzy search through ALL metadata.
To create better linkage, maybe data pages should include below an expandable list of "related datasets", found by "source" attributed.
